### ISSUE TITLE
In document browser, pgs 5-6, Babson docs title end with a comma, has no university

### ISSUE THREAD

**Issue by wehooper** [IUBLibTech/newton_chymistry#69]
On http://carbon.dlib.indiana.edu:8220/search/?page=6&page=5&page=6

the document entry titles end with a comma.

The Huntington Library has an empty \<institution\> element because it's independent.

Can we perhaps produce that comma conditionally if there is content in \<institution\>?

I haven't checked whether Babson was the only case of this as I write, apologies.

**Comment by Conal-Tuohy** [comment]
Here's the Solr metadata generated from one of those docs:
http://carbon.dlib.indiana.edu:8220/solr/ALCH00091/
The title field there is defined in the `search-fields.xml` file:
```xml
	<field name="title"
		label="Title"
		xpath="
			let $id := /TEI/teiHeader/fileDesc/sourceDesc/msDesc/msIdentifier
			return string-join(
				(
					$id/settlement,
					concat($id/collection, ' ', $id/idno),
					$id/repository,
					$id/institution
				),
				', '
			)
		"
	/>
```
The `string-join` function used there should only be including a comma between the items in the sequence, which implies the `msIdentifier` _does_ have an `institution`, but perhaps it's empty?

**Comment by Conal-Tuohy** [comment]
from http://carbon.dlib.indiana.edu:8220/p5/ALCH00091.xml the `institution` is indeed empty:
```xml
<msIdentifier>
	<country key="US">United States of America</country>
	<region key="CA" type="state">California</region>
	<settlement>San Marino</settlement>
	<institution/>
	<repository>Huntington Library</repository>
	<collection>Babson Collection (Burndy Library Collection)</collection>
	<idno>MS. 417</idno>
	<altIdentifier>
		<idno type="collection">Babson</idno>
	</altIdentifier>
	<altIdentifier>
		<idno type="sotheby_lot">63</idno>
	</altIdentifier>
	<altIdentifier>
		<idno type="chadwyck_healey_reel">42</idno>
	</altIdentifier>
	<altIdentifier>
		<settlement>Bloomington</settlement>
		<institution>Indiana University</institution>
		<repository>Digital Library Program</repository>
		<idno type="iunp">ALCH00091</idno>
	</altIdentifier>
	<altIdentifier>
		<idno type="iiif-manifest">https://purl.dlib.indiana.edu/iudl/newton/iiif/ALCH00091</idno>
	</altIdentifier>
</msIdentifier>
```

**Comment by Conal-Tuohy** [comment]
Fixed in 52261aff4efa9d40179d5a911939a25407cd9e7b by ignoring any of the component elements of the `msIdentifier` which are empty.
